packages/vue-instantsearch/src/util/vue-compat/Highlighter/index-vue3.js

```diff
@@ -0,0 +1 @@
+export { default } from '../../../components/Highlighter';
```

Comment:
Because eventually the vue2 will have to go, so we just have to delete what's in that folder.
This is also the rationale I used for `pragma` which only works with Vue3, if I had imported it from `vue-compat` it means we'd have to update the component once we drop support for Vue2.
Not that big of a deal I know but it saves up on some `TODO` comments.